src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,97 @@
+import Text from "src/components/Text";
+import React, { useState } from "react";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+
+import * as S from "./styles";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+    const [name, setName] = useState<string>("");
+    const [subject, setSubject] = useState<string>("");
+    const [content, setContent] = useState<string>("");
+  
+    const handleSubmit = (event: React.MouseEvent) => {
+        event.preventDefault();
```

Comment:
why do we need `preventDefault()`

Reply:
btw this is a question

Reply:
preventDefault is called on the event when submitting the form to prevent a browser reload/refresh. gw gabisa bedain yg mana question yg mana suruh benerin code haha malah gw hapus yg preventDefault